docs/examples/compose/keycloak/compose.yaml

```diff
@@ -0,0 +1,36 @@
+services:
+  kc_postgresql:
+    image: postgres:16
+    healthcheck:
+      test: ["CMD", "pg_isready", "-q", "-U", "keycloak", "-d", "keycloak"]
```

Comment:
Same here, you should use the environment variables provided in the `env.d/kc_postgresql` otherwise once changed this check will not work anymore.